server/api/controllers/article.js

```diff
@@ -580,3 +579,120 @@ export const getArticle = async (req, res) => {
     });
   }
 };
+
+/**
+ * @export
+ * @function updateArticle
+ * @param {Object} req - request received
+ * @param {Object} res - response object
+ * @returns {Object} JSON object (JSend format)
+ */
+export const updateArticle = async (req, res) => {
+  try {
+    const {
+      body,
+      params: {
+        articleId
+      },
+      user: {
+        id: userId
+      }
+    } = req;
+
+    const foundArticle = await Article.findOne({
```

Comment:
@oyedejipeace Remove lines `602` to `615`. They become redundant due to line `616`.